src/Castle.Core/DynamicProxy/Generators/Emitters/ClassEmitter.cs

```diff
@@ -30,13 +30,13 @@ public class ClassEmitter : AbstractTypeEmitter
 		private readonly ModuleScope moduleScope;
 
 		public ClassEmitter(ModuleScope modulescope, String name, Type baseType, IEnumerable<Type> interfaces)
-			: this(modulescope, name, baseType, interfaces, DefaultAttributes, ShouldForceUnsigned())
+			: this(modulescope, name, baseType, interfaces, DefaultAttributes, forceUnsigned: false)
```

Comment:
Why not drop the `forceUnsigned` parameter from the private `CreateTypeBuilder` method?

Reply:
There is one code location where `true` may be passed for `forceUnsigned`:

https://github.com/castleproject/Core/blob/77e76b3c92fa7e244dbd2fb966b75f77859adf8b/src/Castle.Core/DynamicProxy/Generators/InvocationTypeGenerator.cs#L266

(It's part of the logic responsible for placing invocation types in the same module (strong/non-strong named) as the proxy type.)

At least at the surface level, it doesn't look like there is a trivial way of getting rid of that logic.

Reply:
Ah yer, thanks. Merge away!